pkgs/development/python-modules/nose/default.nix

```
@@ -14,14 +13,16 @@ buildPythonPackage rec {
   format = "setuptools";
```

```suggestion
  pyproject = true;
```
and add `setuptools` to `build-system`